src/pages/api/thirdparty/resolvers.ts

```diff
@@ -0,0 +1,33 @@
+import { DateTime } from 'luxon'
+import { GraphQLJSON } from 'graphql-type-json'
+import KeystoneAPI from '../dataSources/keystone'
+
+/* Resolver Types */
+type ArticleInput = {
+  publishedDate: string
+  tag: string
+}
+
+export type ThirdPartyContext = {
+  dataSources: {
+    keystoneAPI: KeystoneAPI
+  }
+}
+
+/* Resolvers */
+export const resolvers = {
```

Comment:
Question: If I'm reading this right tag is a parameter passed in, where would we be confirming that third parties can only request `c-note` since as of now the API is public? Or is that something that is coming soon?

Reply:
GREAT catch. I need to address that.

Reply:
Update: I've changed tag from String to an Enum, and added CNOTE as one of the enum values.